### ISSUE TITLE
youtube dnn loss问题

### ISSUE THREAD

**Issue by cysjtu** [sladesha/deep_learning#11]
https://github.com/sladesha/deep_learning/blob/master/YoutubeNetwork/normal_version/model/model.py

self.loss = tf.reduce_mean(-self.y * tf.log(self.yhat + 1e-24))

这边 y 的值好像是 1/0  ,这个loss是不是有问题

**Comment by ucas010** [comment]
大佬解决了吗？
用的啥数据啊？
有没有啥评价指标啊？
多谢
